src/js/providers/tracks-mixin.ts

```diff
@@ -211,12 +211,13 @@ const Tracks: TracksMixin = {
     disableTextTrack(): void {
         const track = this.getCurrentTextTrack();
         if (track) {
-            // FF does not remove the active cue from the dom when the track is hidden, so we must disable it
-            track.mode = 'disabled';
+            // IOS native captions does not remove the active cue from the dom when the track is disabled, so we must hide it
             const trackId = track._id;
-            if (trackId && trackId.indexOf('nativecaptions') === 0) {
+            if ((trackId && trackId.indexOf('nativecaptions') === 0) || (this.renderNatively && OS.iOS)) {
                 track.mode = 'hidden';
```

Comment:
what does this line do if we set to disabled right after either way?

Reply:
As soon as we set to `hidden`, the text track is hidden on iOS. We want to set to disable as per Rob's comment above:

 `Could we hide the track and then disable it? Leaving a track hidden requires that the track continue to be processed (loading and parsing VTT files... etc.). The only subtitles track that should not be disabled is the one being shown.`